terraform/day0/README.md

```diff
@@ -25,6 +33,8 @@ The `fgcp-ha-ap-lb` module takes a list of subnet names as its `subnets` argumen
 - port3 (nic2) - FGCP hertbeat/sync interface
 - port4 (nic3) - dedicated management interface
 
+Make sure your project quota is high enough to accommodate for the new VPC networks.
```

Comment:
I'll writup the procedure to get your per-project quota increased from 5 (default) to 10 - usually the 2 min AI bot will kick in depending on how you write the quota request (paid services only)